move/operators/sources/operators/operators.move

```diff
@@ -0,0 +1,319 @@
+module operators::operators {
+    use sui::bag::{Self, Bag};
+
+    // -----
+    // Types
+    // -----
+
+    /// The `OwnerCap` capability representing the owner of the contract.
+    public struct OwnerCap has key, store {
+        id: UID,
+    }
+
+    /// The `OperatorCap` capability representing an approved operator.
+    public struct OperatorCap has key, store {
+        id: UID,
+    }
+
+    /// The main `Operators` struct storing the capabilities and operator IDs.
```

Comment:
objects have a max size, so technically use of `vector` is bounded here. Add a comment that this is fine since number of operators in practice will be quite small and below Sui limits.